Controller/CategoryAdminController.php

```diff
@@ -74,7 +75,7 @@ public function treeAction(Request $request)
 
         $rootCategories = $categoryManager->getRootCategories(false);
 
-        if (!$currentContext) {
+        if (!$currentContext && $rootCategories) {
```

Comment:
I think it is not a boolean value. It is probably an array or null, guessing from the function name. A proper emptyness check would be better I think.